### DISCUSSION TITLE
GitHub status checks on a repository.

### DISCUSSION THREAD

**Discussion by tusharrusia** [community/community#60626] [Repositories]
### Select Topic Area

Product Feedback

### Body

I have deployed my project using GitHub repository by installing the cloud hosting website app on my GitHub.

I noticed that on repository page, GitHub shows some checks about rhe deployment.

I don't want these checks to be visible to everyone.

How to hide them ?

https://drive.google.com/file/d/1-PEaPnR-FdBKgY46_zr4nk5KIfy1ASv-/view?usp=drivesdk

**Comment by GhostOf0days** [comment]
That screenshot is unavailable. Could you please instead deploying your website to Netlify, Heroku, or Vercel? Use Netlify for simple use cases/websites and Vercel for more complex, larger use cases/websites.

**Reply by tusharrusia** [reply]
I have updated the screenshot link, could you please answer it now.

**Reply by GhostOf0days** [reply]
You can't. You can make the repo private if you want.

**Comment by jshtuazon2** [comment]
later

**Reply by tusharrusia** [reply]
Can you please elaborate ?

**Reply by GhostOf0days** [reply]
There are some bot accounts in discussions. The person who replied to you is one of them. There is no way to hide the checks. Your best option would be to make the repo private.

**Comment by jshtuazon2** [comment]
yes wait
